### ISSUE TITLE
this should break the build due to formal incorrectnes

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#248]
_From @uweschaefer on 2018-11-23 16:56:38 +0000 UTC_

_Copied from original issue: Mercateo/factcast#248_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2018-11-23 16:59:51 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=h1) Report
> Merging [#248](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/4065c0d2d6b1a597b93bb9824a7fdb2098131e23?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[![Impacted file tree graph](https://codecov.io/gh/Mercateo/factcast/pull/248/graphs/tree.svg?width=650&token=Kt03N0Cztt&height=150&src=pr)](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #248   +/-   ##
=======================================
  Coverage   78.36%   78.36%           
=======================================
  Files          52       52           
  Lines        1368     1368           
  Branches      123      123           
=======================================
  Hits         1072     1072           
  Misses        207      207           
  Partials       89       89
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=footer). Last update [4065c0d...9f1039d](https://codecov.io/gh/Mercateo/factcast/pull/248?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2018-11-23 21:54:05 +0000 UTC_

closes #247
